modules/installation-network-user-infra.adoc

```diff
@@ -21,33 +21,45 @@ addresses and host names to the cluster machines.
 The Kubernetes API server must be able to resolve the node names of the cluster
 machines. If the API servers and worker nodes are in different zones, you can
 configure a default DNS search zone to allow the API server to resolve the
-node names. Another acceptable approach is to always refer to hosts by their
+node names. Another supported approach is to always refer to hosts by their
 fully-qualified domain names in both the node objects and all DNS requests.
 
 You must configure the network connectivity between machines to allow cluster
 components to communicate. Each machine must be able to resolve the host names
 of all other machines in the cluster.
 
 .All machines to all machines
-[cols="2a,5a",options="header"]
+[cols="2a,2a,5a",options="header"]
 |===
 
+|Protocol
+|Port
+|Description
+
+.4+|TCP
 |`2379`-`2380`
 |etcd server, peer, and metrics ports
 
 |`6443`
 |Kubernetes API
 
-|`9000`-`9999`
-|Host level services, including the node exporter on ports `9100`-`9101` and
-the Cluster Version Operator on port `9099`.
-
 |`10249`-`10259`
 |The default ports that Kubernetes reserves
 
 |`10256`
 |openshift-sdn
 
+.3+|UDP
```

Comment:
@wking, do you know what these ports are for? I don't mind adding them, but they need descriptions.